web/create_website.sh

```diff
@@ -0,0 +1,37 @@
+
+# we start by cloning the repository that contains the website
+git clone https://github.com/cernvm/web-team
+git clone https://github.com/cernvm/web-cernvm
+git clone https://github.com/cernvm/web-cvmfs
+
+# we create the structure to hold the html pages
+mkdir content
+pushd content
+mkdir fs filesystem cvmfs appliance vm team
```

Comment:
For the installation on our scratch space, let's forget about the cvmfs, filesystem, and vm aliases